rate-service/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM golang:1.22-alpine
+WORKDIR /app
+COPY .. .
```

Comment:
Why do you copy parent directory with all the services into docker image?

Reply:
Haven't fixed this yet